db/re/item_db_equip.yml

```diff
@@ -137793,6 +138865,56 @@ Body:
       if (.@r>=9) {
          bonus5 bAutoSpell,"SO_PSYCHIC_WAVE",3,100,BF_SHORT|BF_WEAPON,1;
       }
+  - Id: 28761
+    AegisName: Scarlet_Athame
+    Name: Scarlet Athame
+    Type: Weapon
+    SubType: Dagger
+    Buy: 20
+    Weight: 300
+    Attack: 30
+    MagicAttack: 130
+    Range: 1
+    Slots: 1
+    Jobs:
+      Alchemist: true
+      Assassin: true
+      BardDancer: true
+      Blacksmith: true
+      Crusader: true
+      Hunter: true
+      KagerouOboro: true
+      Knight: true
+      Rogue: true
+      Sage: true
+      SoulLinker: true
+      Wizard: true
+    Classes:
+      All_Third: true
+      Fourth: true
+    Locations:
+      Right_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 130
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      skill "WZ_METEOR",1;
+      bonus2 bMagicAtkEle,Ele_Fire,3*.@r;
+      if (.@r>=5) {
+         bonus bMaxHPrate,10;
+         bonus bMaxSPrate,10;
+         bonus bLongAtkDef,5;
+      }
+      if (.@r>=7)
+         skill "WZ_METEOR",min(10,.@r);
+      if (.@r>=9) {
+         bonus bMaxHPrate,10;
+         bonus bMaxSPrate,10;
+         bonus bLongAtkDef,5;
+      }
+      /*Todo Leap Impired State to Enemy 10 Second When Dealing Magic Attack (Add 5 Second of Cooldown to Skill Related on Movement)*/
```

Comment:
Do you plan to implement the status change in this PR?

Reply:
no

i don't know what to do   

XD